packages/concerto-core/lib/decoratormanager.js

```diff
@@ -389,21 +392,21 @@ class DecoratorManager {
                     };
                 })
                 : []);
-        }).flat().filter(i => i.namespace);
+        }).filter(i => i.namespace);
         const { namespaceCommandsMap, declarationCommandsMap, propertyCommandsMap, mapElementCommandsMap, typeCommandsMap }  = this.getDecoratorMaps(decoratorCommandSet);
         const ast = modelManager.getAst(true, true);
-        const decoratedAst = JSON.parse(JSON.stringify(ast));
+        const decoratedAst = rfdc(ast);
```

Comment:
How about `structuredClone`? https://developer.mozilla.org/en-US/docs/Web/API/Window/structuredClone it's a standard function that is also available in NodeJS.

Reply:
https://github.com/davidmarkclements/rfdc/issues/35